### PR TITLE
Fix an error.

### PR DESCRIPTION
**Pull request by mycapatin**
Fix an error.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ you can.
 6. What did you expect to happen?
 
 7. Please provide your cluster manifest. Execute
-  `kops get --name my.example.com -oyaml` to display your cluster manifest.
+  `kops get --name my.example.com -o yaml` to display your cluster manifest.
```
Comment (chrislovecnm) [inline .github/ISSUE_TEMPLATE.md]: does that not work?  I assumed it did

**Comment by chrislovecnm** [comment]
/ok-to-test

Usually, we would want both of these broken apart into different PRs, but I since we have them in different commits, we should be ok.

Thanks!

File: upup/pkg/fi/cloudup/urls.go [hunk 2]
```diff
@@ -66,7 +66,7 @@ func BaseUrl() (*url.URL, error) {
 	} else {
 		kopsBaseUrl, err = url.Parse(baseUrlString)
 		if err != nil {
-			return nil, fmt.Errorf("unable to parse env var KOPS_BASE_URL %q as an url: %v", baseUrlString, err)
+			return nil, fmt.Errorf("unable to parse env var KOPS_BASE_URL %q as a url: %v", baseUrlString, err)
```
Comment (chrislovecnm) [inline upup/pkg/fi/cloudup/urls.go]: If we want to be picky ... url is URL

Comment (mahuihuang) [reply upup/pkg/fi/cloudup/urls.go]: Thanks for your reply. https://itknowledgeexchange.techtarget.com/writing-for-business/which-is-correct-a-url-or-an-url/

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (mahuihuang) [reply .github/ISSUE_TEMPLATE.md]: Sorry ! I think it's more clear, if it's not necessary, i'll remove the commit.

**Comment by justinsb** [comment]
/approve

/lgtm

Thanks @mahuihuang

**Pull request closed without merging**